torch/fx/_symbolic_trace.py

```diff
@@ -305,7 +305,8 @@ def create_arg(self, a: Any) -> "Argument":
         #. Given a Proxy object, return a reference to the underlying IR ``Node``
         #. Given a non-Proxy Tensor object, emit IR for various cases:
 
-            * For a Parameter, emit a ``get_attr`` node referring to that Parameter
+            * For a Parameter, emit a ``get_attr`` node referring to that Parameter,
+              if not exist, create a new ``get_attr`` node and add it to the graph.
```

Comment:
Maybe mention that if it is an external parameter, it will be added as a special attribute similar to non-parameter and non-buffer tensors.

Reply:
fixed